easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'byobu'
+version = '5.133'
+
+homepage = 'https://byobu.org'
+description = """Byobu is an elegant enhancement of the otherwise functional, plain, practical GNU Screen. Byobu
+includes an enhanced profile, configuration utilities, and system status notifications for the GNU screen window
+manager as well as the Tmux terminal multiplexer"""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+builddependencies = [
+    ('Autoconf', '2.69', '', ('GCCcore', '8.3.0')),
+    ('Automake', '1.16.1', '', ('GCCcore', '8.3.0')),
+    ('Autotools', '20180311', '', ('GCCcore', '8.3.0')),
```

Comment:
`Autoconf` and `Automake` are part of the `Autotools` bundle, so no need to include those, and you don't have to hardcode the toolchain either, so please change to:

```python
builddependencies = [
    ('Autotools', '20180311'),
]